package.json

```diff
@@ -20,7 +20,7 @@
     "format": "npm run ci-lint . -- --fix",
     "lint": "npm run ci-lint .",
     "precommit": "lint-staged",
-    "release": "yarn changelog && rollout",
+    "release": "rollout",
```

Comment:
no more changelog?

Reply:
this wasn't working right i'm gonna follow up with something in rollout